src/mlagility/cli/cloud/setup_part_1.sh

```diff
@@ -0,0 +1,53 @@
+# Setup
+
+MYUSER=azureuser
+MYHOME=/home/$MYUSER
+
+# Install NFS
+
+# sudo apt-get -y update
+# sudo apt-get -y install nfs-common
+# sudo mkdir -p /mount/mlagilitystorage/mla-share
```

Comment:
Remove dead code

Reply:
Fixed in https://github.com/groq/mlagility/pull/298/commits/6b4a178790065ac7e312c24618bbebd69009059e